### DISCUSSION TITLE
Blueprints

### DISCUSSION THREAD

**Discussion by ssiwm** [gameserverapp/Platform#906] [Suggestions]
Can you create an nsa channel on Discord where we can place an order for blueprints? Or indicate interest in a specific game, maybe then something will start happening there. Personally, I lack the skills and knowledge but I really need blueprints for at least 1 game :)

**Comment by birdman3131** [comment]
https://discord.com/channels/315049169071112193/1249788070648090646 would probably be the best spot for now.
